invenio_rdm_records/services/pids/tasks.py

```diff
@@ -8,17 +8,22 @@
 """RDM PIDs Service tasks."""
 
 from celery import shared_task
+from invenio_db import db
+from invenio_access.permissions import system_identity
 
 from invenio_rdm_records.proxies import current_rdm_records
 
 
 @shared_task(ignore_result=True)
-def register_pid(pid, scheme, record_id, provider_name=None):
+def register_pid(pid_type, pid_value, record_id, provider_name=None):
```

Comment:
naming: wondering if we should call it `recid` instead of `record_id`, what do you think?